lib/routes.php

```diff
@@ -15,9 +15,12 @@
 
 // /{pagename}.{representation}
 if ($representation != '') {
-  if (!$page_exists) {
+  if (!$page_exists) {	  
```

Comment:
There’s some extra whitespace here.

```suggestion
  if (!$page_exists) {
```

Although it doesn’t change the result, it adds some noise in the pull requests. You can configure your text editor to remove trailing whitespace automatically.